templates/master.yaml.tmpl

```diff
@@ -3228,6 +3291,17 @@ systemd:
       ExecStart=/opt/k8s-addons
       [Install]
       WantedBy=multi-user.target
+  - name: debug-tools.service
+    enabled: true
+    contents: |
+      [Unit]
+      Description=Calicoctl, etcdctl and crictl tools
+      After=network.target
+      [Service]
+      Type=oneshot
+      ExecStart=/opt/install-debug-tools
+      [Install]
+      WantedBy=multi-user.target
```

Comment:
do we need to install it automatically? from a security perspective, it  would be best to just have an install file that we can run on demand, but that just opinion, not a blocker

Reply:
That's an additional context you should know. But also no strong opinion on this